python/paddle/nn/layer/transformer.py

```diff
@@ -278,8 +278,8 @@ def compute_kv(self, key, value):
                 should be float32 or float64.
 
         Returns:
-            tuple: A tuple including transformed keys and values. Their shapes \
-                both are `[batch_size, num_heads, sequence_length, embed_dim // num_heads]`, \
+            tuple: A tuple including transformed keys and values. Their shapes
+                both are `[batch_size, num_heads, sequence_length, embed_dim // num_heads]`,
                 and their data types are same as inputs.
```

Comment:
```suggestion
            Tuple. A tuple including transformed keys and values. Their shapes
            both are `[batch_size, num_heads, sequence_length, embed_dim // num_heads]`,
            and their data types are same as inputs.
```
除了参数外的其他描述，同一段落的内容都需要用相同的缩进，否则会如下所示
<img width="854" alt="image" src="https://github.com/PaddlePaddle/Paddle/assets/70642955/c4e20b63-b1d7-4432-8815-1dd3236523e7">

Reply:
Done